src/QueryBuilder.php

```diff
@@ -17,6 +17,9 @@ class QueryBuilder
    */
   protected $cache;
 
+  /** @var bool $prettyPrint State of using JSON_PRETTY_PRINT for writing the JSON Files */
+  protected bool $prettyPrint = false;
```

Comment:
Type declarations are supported in PHP >= 7.4

Reply:
See https://github.com/rakibtg/SleekDB/pull/234#discussion_r878758462